### PR TITLE
Date format

### PR DESCRIPTION
**Pull request by langeyouxiang**
1.Add set the date format in the settings
2.Add date format: MM/dd

### CODE REVIEW

File: source/views/DateView.mc [hunk 1]
```diff
@@ -13,7 +14,8 @@ class DateView extends Components.Box {
         var posY = self.getPosY();
         var width = self.getWidth();
 
-        var dateObj = formatDateToEng(Time.now());
+        var getDateFormat = SettingsModule.getValue(SettingsModule.SettingType.DATE_FORMAT);
```
Comment (shbumba) [inline source/views/DateView.mc:17]: Could you please update the way you get and save setting as in this commit https://github.com/shbumba/SimplePixels/commit/10695146e30991df85a176e0e6f48b35a05b3258#diff-56676d6adbb568c19d7318b283ca7174b643c7884ef7c1520dbc5da0dd534b26R30

The thing is that getting settings from Properties every time causes parsing of values from file storage, which causes excessive load on older devices

File: source/views/settingsMenu/SettingsMenu.mc [hunk 2]
```diff
@@ -63,6 +63,13 @@ function SettingsMenu() as WatchUi.Menu2 or WatchUi.CustomMenu {
             :label => Rez.Strings.SecondTimeFormat
         })
     );
+    menu.addItem(
+        SettingsMenuBuilder.generateMenuItem(SettingsMenuBuilder.ITEM, {
+            :identifier => SettingType.DATE_FORMAT,
+            :label => Rez.Strings.DateFormat
+            // :subLabel => _getSubLabelValue(SettingType.DATE_FORMAT)
```
Comment (shbumba) [inline source/views/settingsMenu/SettingsMenu.mc:70]: I think we can uncomment this :)

File: source/utils/formatDateToMMdd.mc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Toybox.Lang;
+import Toybox.Time;
+import Toybox.Time.Gregorian;
+
+function formatDateToMMdd(time as Time.Moment) as Array<String> {
```
Comment (shbumba) [inline source/utils/formatDateToMMdd.mc:5]: There is a problem here which is already solved in the `formatDateToEng` function. The thing is that `date.month.toString()` can return a string in different languages, which is not supported by the font we use, which leads to a broken dial, for this we will have to add localization and change fonts. So for now I recommend to do the following, we need to do a small refactoring of the `formatDateToEng` function and make it reusable in your function. I would do it this way, create a `formatDate` function that would return an array with day and month, `formatDateToEng` would take the values from it and concatenate them into a string in `DD MM` format, your function would do the same thing but concatenate the data into a string as `DD MM`.

**Review by shbumba — CHANGES_REQUESTED** [review]

File: source/utils/formatDateToMMdd.mc (hunk 3 above)
Comment (langeyouxiang) [reply source/utils/formatDateToMMdd.mc:5]: Thank you very much for your correction. I will complete the revision on Sunday (UTC+8).

File: source/views/DateView.mc (hunk 1 above)
Comment (langeyouxiang) [reply source/views/DateView.mc:17]: OK.

File: source/views/settingsMenu/SettingsMenu.mc (hunk 2 above)
Comment (langeyouxiang) [reply source/views/settingsMenu/SettingsMenu.mc:70]: Hahahaha! I was going to show the formatting already set below Data format, but then I realized I needed to refactor a lot of code, so I commented it out for now. I‘ll remove this comment and add it back later.

**Pull request closed without merging**